src/lib/viewers/BaseViewer.js

```diff
@@ -931,17 +931,29 @@ class BaseViewer extends EventEmitter {
             return;
         }
 
-        const boxAnnotations = this.options.boxAnnotations || new global.BoxAnnotations(viewerOptions);
+        const boxAnnotations =
+            this.options.boxAnnotations || new global.BoxAnnotations(viewerOptions, intl.createAnnotatorIntl());
         this.annotatorConf = boxAnnotations.determineAnnotator(this.options, this.viewerConfig);
 
         if (!this.annotatorConf) {
             return;
         }
 
-        const annotatorOptions = this.createAnnotatorOptions({
+        const options = {
             annotator: this.annotatorConf,
             modeButtons: ANNOTATION_BUTTONS,
-        });
+        };
+
+        if (this.areNewAnnotationsEnabled() && boxAnnotations.getOptions) {
+            const annotationsOptions = boxAnnotations.getOptions();
+
+            if (annotationsOptions) {
+                const { locale, messages } = annotationsOptions;
```

Comment:
This seems a little confusing. Is `language` omitted intentionally? Can we simplify the options logic?

```
const { intl } = boxAnnotations.getOptions ? boxAnnotations.getOptions() : {};
const annotatorOptions = this.createAnnotatorOptions({
    annotator: this.annotatorConf,
    intl: intl || intlUtil.createAnnotatorIntl(),
    modeButtons: ANNOTATION_BUTTONS
});
```

Reply:
Sure, I passed it to the boxAnnotations instantiation on purpose to be explicit how it should be used.  This should be fine as well. Thanks for the input